02week/bio/css/style.css

```diff
@@ -1,48 +1,46 @@
-a:hover{
-  color:white;
+a:hover {
+  color: white;
 }
 
 .img1 {
```

Comment:
Your classes `.img1` and `.img2` are too generic. Make sure that you name your classes more semantic like `.profile-picture` or `.state-logo`